index.html

```diff
@@ -221,8 +355,30 @@ <h3 class="project_heading">Professional Art Printing Data</h3>
           </ul>
         </div>
 
-        <button class="see_project">See Project</button>
+        <button class="popup-btn see_project">See Project</button>
       </section>
+      <div class="popup dn">
+        <div class="popup-container">
+          
+          <h1 class="popup-heading"></h1>
+          <img class="close-btn" src="#" alt="close button" />
+          <ul class="popup-list">
+            <li class="popup-list-item"></li>
+            <li class="popup-list-item"></li>
+            <li class="popup-list-item"></li>
+          </ul>
+          <img class="popup-img" src="#" alt="project image" />
+          <div class="popup-buttons">
+             <p class="popup-description"></p>
+            <button class="see-live button">
+              See live <i class="fas fa-external-link-alt"></i>
+            </button>
+            <button class="see-source button">
+              See source <i class="fab fa-github"></i>
+            </button>
```

Comment:
-  [ ] As it is a requirement for this project(`see screenshot below`), kindly add the demo links(`source and live`) as part of the objects in the JavaScript projects array, This will make adding projects in the future easier, as you will have to add everything in one place, as another object in the projects array

![array](https://user-images.githubusercontent.com/62849872/151458954-488d66a0-88fb-4c7d-8ac3-878555ee7dd6.png)
.